satellite/peer.go

```diff
@@ -268,10 +268,12 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config, ve
 		}
 	}
 
+	overlayCache := overlay.NewInternalDBCache(peer.DB.OverlayCache(), config.Overlay.UptimeFlushInterval)
```

Comment:
Add as peer.Overlay.XYZ.

Reply:
Do you mind to explain why we should add as above?